tests/README.md

```diff
@@ -41,6 +41,7 @@ Use an isolated VM since it's constantly occupying +2 threads simultaneously
 ```
 source .virtualenv/bin/activate
 nohup ./script.sh &
+nohup ./script.sh >/dev/null 2>&1 &
```

Comment:
I think I'm misunderstanding something with the instructions here. Can you explain why the readme is stating to run the same script (`script.sh`) twice?

- I understand the first call (L43) is running `script.sh` with no-hangup, and doing so in the background.
- And I understand that the second call (L44) is doing the same thing except discarding anything output to stdout.

But I'm not following why we run the same script, `script.sh`, twice? 

Or is the intent that you'd run `script1.sh` and `script2.sh` at the same time?

Reply:
The instructions are misleading. I'm updating it now